alembic/versions/23193b41cfea_added_got_staff_merch.py

```diff
@@ -34,7 +34,11 @@ def sqlite_column_reflect_listener(inspector, table, column_info):
 
 
 def upgrade():
-    op.add_column('attendee', sa.Column('got_staff_merch', sa.Boolean(), server_default='False', nullable=False))
+    if is_sqlite:
+        with op.batch_alter_table('attendee', reflect_kwargs=sqlite_reflect_kwargs) as batch_op:
+            batch_op.add_column(sa.Column('got_staff_merch', sa.Boolean(), server_default='False', nullable=False))
+    else:
+        op.add_column('attendee', sa.Column('got_staff_merch', sa.Boolean(), server_default='False', nullable=False))
```

Comment:
Whoops - I totally forgot to do the sqlite checking last time I did migrations as well.  Sorry about that - I swear I'll remember next time!